requirements/base.txt

```diff
@@ -26,5 +26,7 @@ pytz==2015.2
 South==1.0.1
 voluptuous==0.8.5
 
+-e git+https://github.com/edx/nltk.git@2.0.6#egg=nltk
```

Comment:
Since this is also needed for AI grading, let's move this down below the `#AI grading` comment.